tsconfig.json

```diff
@@ -0,0 +1,19 @@
+{
+  "compilerOptions": {
```

Comment:
This should also include `"strictNullChecks": true`, otherwise the linting is unhappy:

~~~~
> oauth2-server@5.0.0-dev.0 lint:all ..../node-oauth2-server
> tslint -p tsconfig.json -c tslint.json

strict-type-predicates does not work without --strictNullChecks
~~~~